iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -196,6 +196,18 @@ using ConfigFilePathString_t = cxx::string<1024>;
 using ProcessName_t = cxx::string<MAX_PROCESS_NAME_LENGTH>;
 using NodeName_t = cxx::string<100>;
 
+namespace mepoo
+{
+using SequenceNumber_t = std::uint64_t;
+using BaseClock_t = std::chrono::steady_clock;
+
+// use signed integer for duration;
+// there is a bug in gcc 4.8 which leads to a wrong calcutated time
+// when sleep_until() is used with a timepoint in the past
+using DurationNs_t = std::chrono::duration<std::int64_t, std::nano>;
+using TimePointNs_t = std::chrono::time_point<BaseClock_t, DurationNs_t>;
```

Comment:
these could probably also fit into the plain `iox` namespace. It's a bit weird to have a `DurationNs_t` which is not based on `cxx::Duration` though

Reply:
We should remove them completely and should always use `units::duration`.

Reply:
so, no chrono based timepoints?

Reply:
Is the issue in cxx::Duration with the rounding error resolved? For example when you use 1 ns then its represented as zero in the duration.

Reply:
Not, it's still using a `double`. @elfenpiff will receive the honour I suppose :P